### PR TITLE
Controls for admitting / removing reset users on teams

### PR DESCRIPTION
**Pull request by buoyad**
- ~Have frontend detect duplicate entry (PUK-less invite and reset user) in `teamList` and reduce to just the PUK-less invite.~ nevermind, core will do this

@keybase/react-hackers there is some edge behavior that is being worked on in the service. In the end though, we should be calling `addMember` and `removeMember` for the reset user flow, which this PR does.

cc @mmaxim

### CODE REVIEW

**Comment by buoyad** [comment]
After resetting `dannytest3` and calling `addMember(dannytest3)`, `list-members` shows this:
image

**Comment by buoyad** [comment]
`dannytest3`: gets `multiple errors while loading team list`and `error unboxing chat message` for the re-admitted team. 
`ayoubd` (admin): everything looks normal once `dannytest3` provisions a device

**Comment by buoyad** [comment]
log from `dannytest3`'s side: 06e828bd4df84b81ff6f901c

File: shared/constants/selectors.js [hunk 1]
```diff
@@ -30,6 +30,16 @@ const userIsInTeam = (
   return teamNameToMemberUsernames.getIn([teamname, username])
 }
 
+const userIsActiveInTeam = (
+  {entities: {teams: {teamNameToMembers}}}: TypedState,
+  teamname: string,
+  username: string
+) => {
+  const members = teamNameToMembers.get(teamname, Set())
+  const member = members.find(mem => mem.username === username)
```
Comment (chrisnojima) [inline shared/constants/selectors.js]: not ideal that this is an object which basically makes it being in the set and us always doing finds O(N)

File: shared/teams/team/member-row/index.js [hunk 2]
```diff
@@ -35,48 +47,61 @@ export const TeamMemberRow = (props: Props) => {
...
-          </Text>
+          </Box>
+          <Box style={{...globalStyles.flexBoxRow, alignItems: 'center'}}>
+            {!!fullName &&
+              active && (
+                <Text style={{marginRight: globalMargins.xtiny}} type="BodySmall">
+                  {fullName} •
+                </Text>
+              )}
+            {!!active &&
```
Comment (chrisnojima) [inline shared/teams/team/member-row/index.js]: lets not have complicated logic like this in jsx. instead hoist this up into its own var
```
const crown = active && type &&...
```
then just
```
{crown}
```

Comment (chrisnojima) [reply shared/teams/team/member-row/index.js]: i think if its a simple single check its fine but if its bigger than that likely too noisy

File: shared/constants/selectors.js (hunk 1 above)
Comment (chrisnojima) [reply shared/constants/selectors.js]: we don't have to change this here, but noting this is a code smell

**Pull request merged**